core/shared/src/test/scala/cats/parse/RadixNodeTest.scala

```diff
@@ -25,7 +25,7 @@ import org.scalacheck.{Gen, Prop}
 import org.scalacheck.Prop.forAll
 
 class RadixNodeTest extends munit.ScalaCheckSuite {
-  val tests: Int = if (BitSetUtil.isScalaJs) 50 else 20000
+  val tests: Int = if (BitSetUtil.isScalaJvm) 20000 else 50
```

Comment:
Same here.

Reply:
It takes about 5 times longer to run this test(RadixNodeTest) on native than on JVM. Is this acceptable speed?

Reply:
Wow i wasn’t expecting such a difference. Out of scope and just curious … any clue to where this degradation is coming from?

Reply:
Sadly I'm also hitting big performance deficits in https://github.com/typelevel/spire/pull/1180. In some cases Scala.js seems to be 20x faster than Scala Native.